### ISSUE TITLE
Best metrics on im2latex-100K?

### ISSUE THREAD

**Issue by Sinon02** [kingyiusuen/image-to-latex#4]
Hey guy, your work is cool! If possible, can you tell me the best metrics you got when training on im2latex-100K, such as BLEU WER , ROUGE, etc.?

**Comment by kingyiusuen** [comment]
My best attempt has a character error rate (CER) of 0.17 on the test set. With some hyperparameter tuning and better data cleaning, I'm sure you can get a better result!

**Comment by Sinon02** [comment]
> My best attempt has a character error rate (CER) of 0.17 on the test set. With some hyperparameter tuning and better data cleaning, I'm sure you can get a better result!

Thanks a lot!
